packages/scripts/package.json

```diff
@@ -10,6 +10,7 @@
     "directory": "packages/scripts"
   },
   "devDependencies": {
-    "clang-format": "1.2.4"
+    "clang-format": "1.2.4",
+    "just-scripts": "^0.36.1"
```

Comment:
We really don't need just here for now, but it doesn't hurt anything, and we might want to do some tasks eventually.